packages/e2e/features/ui/components/storage/storage-image/load-public-images-from-two-s3-buckets.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Load two images, each from a different S3 bucket with public access level settings
+
+  Background:
+    Given I'm running the example "ui/components/storage/storage-image/multi-bucket"
+
+  @react
+  Scenario: I successfully load two images from two buckets
```

Comment:
public cat 1 is from the first bucket and public cat 2 is from the second bucket